velox/functions/prestosql/tests/ProbabilityTest.cpp

```diff
@@ -155,5 +155,27 @@ TEST_F(ProbabilityTest, binomialCDF) {
       binomialCDF(-2, 0.5, -1), "numberOfTrials must be greater than 0");
 }
 
+TEST_F(ProbabilityTest, invBetaCDF) {
+  const auto invBetaCDF = [&](std::optional<double> a,
+                              std::optional<double> b,
+                              std::optional<double> p) {
+    return evaluateOnce<double>("inverse_beta_cdf(c0, c1, c2)", a, b, p);
+  };
+
+  EXPECT_EQ(0.0, invBetaCDF(3, 3.6, 0.0));
```

Comment:
Please add tests with combinations of std::nullopt, kDoubleMax, kDoubleMin and kNan also.

Reply:
Addressed in latest commit with exception to the kDoubleMax, kDoubleMin. Unable to test those due to Boost assert.